packages/nx-sonarqube/src/executors/scan/schema.json

```diff
@@ -65,6 +65,11 @@
       "description": "Add more detail to both client and server-side analysis logs",
       "type": "boolean",
       "default": false
+    },
+    "executor": {
```

Comment:
This option can be removed and have it auto sense for the `jest` or `vite` option.